frontend/src/components/elements/DeckGlChart/DeckGlChart.tsx

```diff
@@ -25,10 +25,11 @@ import DeckGL, {
   ScatterplotLayer,
   ScreenGridLayer,
   TextLayer,
+  // @ts-ignore
```

Comment:
There is an existing types for deck.gl , you can see it in package.json under devDependencies as "@danmarshall/deckgl-typings", sometimes this raises an exception but if you delete node_modules folder and install it again using `yarn` will be solved.

The problem is that `make all-devel` is not cleaning node_modules/ folder.

Reply:
Will get rid of it.